### ISSUE TITLE
Are these in your geneology

### ISSUE THREAD

**Issue by judyfong** [DuncanAdamson/duncanadamson.github.io#3]
https://mathshistory.st-andrews.ac.uk/Biographies/Danielsson/

https://en.wikipedia.org/wiki/Paul_Erd%C5%91s

I haven't had the chance to look at your geneology report

**Comment by judyfong** [comment]
Also it would be useful to add birthday and year died in your mathematical genealogy

**Comment by DuncanAdamson** [comment]
> https://mathshistory.st-andrews.ac.uk/Biographies/Danielsson/
> 
> https://en.wikipedia.org/wiki/Paul_Erd%C5%91s
> 
> I haven't had the chance to look at your geneology report

Lol I wish but no

**Comment by DuncanAdamson** [comment]
> Also it would be useful to add birthday and year died in your mathematical genealogy

Yeah, but I just stole this from https://www.mathgenealogy.org/, so until they can put in the work for me, it will have to remain very basic

**Comment by judyfong** [comment]
Ah. Ok time for me to check if their code is opensource then.

**Comment by DuncanAdamson** [comment]
I just made a web scraper (attached as a zip because github rightfully doesn't trust me
[scraper.py.zip](https://github.com/user-attachments/files/16818785/scraper.py.zip)
)

**Comment by judyfong** [comment]
If you created it as a gist then it'd be easily shareable. Like here: https://gist.github.com/judyfong/8f59e79fdedfed16aa7b5d8129d1f99e

**Comment by DuncanAdamson** [comment]
Cool, didn't know about that
